### ISSUE TITLE
How to select only single line in vertically 

### ISSUE THREAD

**Issue by NiravPotenza** [MFlisar/DragSelectRecyclerView#8]
please help me to get only vertical item selection like this using drag only 
![1579528493996](https://user-images.githubusercontent.com/47742335/72731917-a962c600-3bba-11ea-9f90-3896c6def8d6.JPEG)

**Comment by NiravPotenza** [comment]
@MFlisar can u please guide me for the same

**Comment by MFlisar** [comment]
This is not part of my library. You can try to add it yourself.

**Suggestion**
Implement the logic inside your code yourself in this line:
https://github.com/MFlisar/DragSelectRecyclerView/blob/9d8b02e89568ab5507564626f1231d82babe1e5a/demo/src/main/java/com/michaelflisar/dragselectrecyclerview/demo/MainActivity.java#L77

Save the first position in this callback (if calledFromOnStart == true) and implement your logic yourself (only select the items between start and end which columns are the same as the one from the first item)

**Comment by NiravPotenza** [comment]
yes i m trying for the same but can u please let me know why it's not working in **nested scrollview**
